source/agg-pipeline-builder/maxtime-ms-pipeline.txt

```diff
@@ -0,0 +1,66 @@
+.. _set-maxtime-ms-agg:
+
+=======================================
+Set Max Time MS for Aggregation Queries
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use the :guilabel:`Max Time MS` option on the :guilabel:`Aggregations` tab to 
+specify an upper time limit in milliseconds for aggregation pipelines
+that run in |compass|. 
+
+About This Task 
+---------------
+
+By default, :guilabel:`Max Time MS` is set to 60000 milliseconds, or 60 seconds. Consider 
+raising this value if you have a large collection or your operations frequently 
+time out. You can also consider creating :ref:`indexes <compass-indexes>` to 
+improve query performance.
+
+If you already specified a ``maxTimeMS`` value in the :ref:`Compass Settings 
+panel <compass-interface-settings>`, the :ref:`command line <cli-options>`, or 
+a :ref:`configuration file <config-file-options>`, the value you enter for 
+:guilabel:`Max Time MS` value must be lower than the limit you previously 
+specified.
+
+If your aggregation operation goes over the time limit, |compass-short| raises 
```

Comment:
[question] Should this sentence go at the top of "About this Task"? I'm not 100% sure, but it seems natural to talk about default settings and such before going into the behavior.

Reply:
I think the ordering you have right now flows nicely. I'd leave as-is.